commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -30,7 +34,18 @@ def run_filter(self, request):
         sku = request.query_params.get('sku', '').strip()
         course_run = request.query_params.get('course_run_key', '').strip()
 
-        if is_redirect_to_commercetools_enabled_for_user(request) and course_run:
+        ct_api_client = CommercetoolsAPIClient()
+
+        if course_run:
+            try:
+                commercetools_available_course = ct_api_client.get_product_variant_by_course_run(course_run)
+            except HTTPError as exc:
+                logger.exception(
+                    '[get_product_variant_by_course_run] Failed to get CT course for course_run: %s', course_run
+                )
+                raise APIException("Error while fetching course variant from CT") from exc
```

Comment:
This Exception raised is not being caught anywhere in the endpoint from where this pipeline is being initiated.

Ref: https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/lms/views.py#L48

Reply:
Working on this unit test as we speak